source/wrangling.md

```diff
@@ -754,7 +581,7 @@ lang_home_tidy
 ```
 
 ```{code-cell} ipython3
-lang_home_tidy.dtypes
+lang_home_tidy.info()
```

Comment:
holy crap this is a useful function!

We should absolutely teach `.info()` earlier. How about in a new section right before "Tidy data" (i.e., at the end of the "Data frames, series, and lists" section)

Reply:
(and of course explain all the printouts you get with `.info()`

Reply:
added lines 655 and following to explain this. It doesn't seem like it needs its own heading? but happy to move this and create a new section if you prefer